PARTNERS.md

```
@@ -59,7 +59,7 @@ to their target OpenShift version
     repository.  The next release will be available for install via the `oadp-dev` 
     branch until such time the next release branch is created, the `oadp-1.<version>+1` 
     branch will be made available for install.
-     - latest branched version is [oadp-1.4](https://github.com/openshift/oadp-operator/tree/oadp-1.4)
+     - latest branched version is [oadp-1.6](https://github.com/openshift/oadp-operator/tree/oadp-1.6)
```

markdownlint-cli2 (0.23.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 5

(MD005, list-indent)